contrib_bots/lib/xkcd.py

```diff
@@ -0,0 +1,120 @@
+import requests
```

Comment:
Conventionally there is an expected grouping of Python imports: standard library imports, related third party imports and local application/library specific imports:

https://www.python.org/dev/peps/pep-0008/#imports

Also having them in alphabetical order will look good too.